scripts/editor.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+# We have to use npm instead of yarn because it causes trouble in netlify
+
+# Link local mermaid to npm
+pushd packages/mermaid || exit
+npm link
+popd || exit
```

Comment:
Very likely it is not necessary at all, but exit have 0 as default status. If any other script on top of that is relying on that, it can be misleading, because it masks its behavior:
```
./editor.sh && echo "success"
```
If it is exactly what is needed let it be as it is.

Just in case, `$?` returns exit code of previous command

Reply:
The `|| exit`s were auto-added by my shellcheck extension. I've removed them based on @aloisklink's suggestion.